### ISSUE TITLE
Balance of Wanderer Mind Missions Seems Too Hard

### ISSUE THREAD

**Issue by SimulatedKnave** [endless-sky/endless-sky#7896]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Not, strictly speaking, a bug as such. But certainly not a feature request.  

I am struggling to see how this mission is supposed to be balanced with a single ship. I have made it to the Wanderer mission where you get the Mind installed in the space station. 

Surviving after doing that appears to be borderline impossible. 

Notably, surviving getting to the station was extremely difficult, even with the Wanderer backup. But I eventually managed to get in with no losses. Then we try to get out. 

Literally impossible. The only difference is how fast we all get wiped out. The enemy doesn't even appear to take noticeable losses. 

I eventually managed to run away (without losses), and would try jumping in again. Comparing the enemy count to the files, the Wanderers appear to have managed to take out maybe one or two ships before being completely annihilated. I eventually got sick of it and cheated myself a bunch of Derechos to see what it would take (and to ensure that it wasn't just that I can't build a ship worth a damn). 

Experiments reveal that 25 stock Derechos is enough to win. With some losses. 20 gets wiped. 

I am 99% sure that that's not balanced properly for a single ship plus the allied Wanderer support (who, tbf, at this point are very dead). 

Looking at the files, the numbers just seem wildly uneven. The friendly forces are smaller than the enemy forces, and that's before you add in the 512s the enemy gets that the allies have no real equivalent for. Hell, I've had the Wanderer fleet take serious losses ON THE WAY THERE. 

And this is with me retrofitting in hyperdrives so we all show up together. Showing up with jump drives usually sees two or three ships focus-deleted within ten seconds, and things quickly get worse from there. 

The previous Kor Mereti battle over the Kor Efreti planet saw the Kor Mereti easily disabling eight Quarg Wardragons on one of the times I loaded. I'm not sure what's going on, but I feel like the intended balance is not being achieved. 

### Steps to Reproduce

Load the attached save and see if you want. It will be added momentarily. 

### Expected Behavior

Not instant death? Or, at least, not instant death in a battle I'm apparently expected to be able to win with one ship, or at least with a reasonable fleet? 

Also, it is sort-of unfair to have the fleet on the way in be not persistent. I feel for anyone who goes to the trouble of actually shooting them down. 

Honestly, I'm not entirely clear why the Wanderers need to stick around. The Mereti appear to want to keep the Mind (based on their behaviour), and it'd be perfectly appropriate to flee from the overwhelming enemy and come back the next day to see what happened. 

### Link to save file

[Melchior Jack.txt](https://github.com/endless-sky/endless-sky/files/10252195/Melchior.Jack.txt)

### Operating System

Windows 10. 

### Game Source

Steam

### Game Version

0.9.16.1

**Comment by warp-core** [comment]
Do you have a save from before starting the mission?

The fight is certainly difficult, but the difficulty should not be in getting to the station initially. I've never had much difficulty doing this.
Various changes over the years have made the defence component of the mission perhaps too difficult, but I don't think your ship is really built very well for fighting Mereti, or anyone really. Your power situation is especially bad, you won't get very far into the fight before you run out of energy. The ship also doesn't have a lot of health, but this can be mitigated if you avoid becoming the target of too many enemy vessels, this is best done by sticking near allies and focusing your weapons on ships your allies are already targeting. Your ship also doesn't really do a whole lot of damage, it's not nothing, but it's well below the capabilities of similarly sized ships in this fight.
I am attempting some of these fights with your build, and it's not doing very well in even the first fight against the Mereti (defending Spera Anatrusk). I am heavily relying on the Wanderers to actually defeat the enemy ships here and not really contributing much.

**Comment by SimulatedKnave** [comment]
Unfortunately not re the save. 

Re the build: I very rarely run out of energy in battle (bar lots and lots and lots of ion damage). It's a little hard to tell whether my escorts are, but they don't seem to be. I loaded up as them just now (and left the Dredgers on the planet), tried fighting some random Mereti - we died (once the 512 and about ten escorts jumped in), but we didn't run out of energy while doing it (and they had numbers on their side in any case). Frankly, I usually let my escorts do most of the fighting - it seems necessary to have escorts for survivability, and if I do have them it makes more sense for me to do all the stuff they can't do (like boarding) and they do that. 

Try taking one of the Derechos near the top into combat instead, if that's your concern. Those are the standard escorts I'm using. The ones above the Shield Beetles. 

Also "not a lot of health" is "more than every other ship currently available to me EXCEPT the the Derecho." Unless I've missed one somewhere. 

In any case, potential build weaknesses are why I tested the problem by throwing lots of stock Derechos at it. Like I said...needing 25 of them seems like a lot. And like I said, the fleet over the Kor Efreti world was taking out eight Wardragons quite regularly. If the Quarg are struggling with the Mereti, I don't know that my build quality is what's making the difference.

**Comment by SimulatedKnave** [comment]
I do actually have a save from before the mission. Sorry about that. Attached. 

[Melchior Jack~3026-11-07.txt](https://github.com/endless-sky/endless-sky/files/10252415/Melchior.Jack.3026-11-07.txt)

You'll have to go and pick up the Wanderer fleet first, mind you.

**Comment by warp-core** [comment]
Oh, you were bringing your own Derechos as well?
One reason you might be having difficulty getting to the station is that your ship only has a jump drive. This means that you will not enter the system clustered around the rest of the fleet, may arrive several seconds before them, and will not be moving very much upon entering the system.
This will be exacerbated if you use the default route chosen when you select the destination system from Kaliptari as it will direct you through Fornarep. The mission escorts will follow you there, but then be unable to follow you directly to Chimitarp, having to traverse Mekislepti first. This will mean they are even futher behind you, and your own ship is not strong enough to last long against a significant Mereti force.
If you have a hyperdrive, you will be able to arrive at the same time as the escorts more easily, as well as entering the system from the same direction. Perhaps most importantly, you will already be moving towards the inner system, largely towards your target, instead of needing to turn significantly and then accelerate from a standstill.

The introduction of arrival distances most likely made this part more difficult, but using a hyperdrive, I've never really had much difficulty reaching the station and, even using your save, by planning my route carefully and waiting for the other ships, I was able to land without nearly as much difficulty as you did, there is some luck involved due to the random arrangement of Mereti ships in the system initially, but my first attempt only had two Derechos disabled upon landing and, in the second, I was able to make it to the station without losing a single ship. (Having my escorts gather around me once entering the system helped, I think. It keeps them from trying to engage the enemy and getting themselves swarmed, the other ships all arrived together and can work together to survive long enough for me to make it to the station.)

The final fight itself is perhaps more variable. The first time, it didn't go so well, though this is perhaps because I was trying to play your flagship the way I would play my own, which is not a good idea because your ship is not built like my own. The second time, however, I chose to direct my three Derechos instead of trying to fight with the flagship. This went much better. One of the Derechos was lost, but several of the Wanderer ships survived the battle and all the enemy ships were defeated.

**Comment by SimulatedKnave** [comment]
Originally, I went in with the flagship, three Derechos, another Dredger (which I stopped taking because it died constantly), and the three Shield Beetles. I can get into the station. I don't think I ever approached from the default route, as it's nice to have a save right before the battle and there's a nice empty planet on the system to the southwest to give you that. As I'd noted above, I had to switch to hyperdrives, but could do it. Careful route planning can only do so much about the swarm of Mereti, since the station is often quite close to where they spawn (and if I let them fight the Wanderers too long, I start losing Wanderers, which carries longer term problems). 

The final fight...I dunno. You're almost certainly better at this than me, but I find it goes badly pretty much no matter what. If I do the usual thing of "pull back, group fleet, engage enemies prudently and with weight of numbers, the Wanderers get wiped and the enemy fleet that survives is big enough to easily annihilate mine. If I stay in the combat, the Wanderers AND I get wiped. 

Are we both playing the same version? 

(And that is basically how I play flagships now. I turned autofire on, discovered it can do things like "not shoot the forward weapons when there's nothing in front of the ship" and at that stage there's no sense in me doing anything other than flying and directing the fleet. I just picked up the Emerald Sword in my "before all this" save, and once I get it outfitted a bit more will try with that as the flagship).

**Comment by SimulatedKnave** [comment]
Tried again with the Emerald Sword. Lost a Shield Beetle on the way in (because "don't fly into the enemy" is apparently too complicated an order), but everything else survived and I ended up with a result much like you describe. Didn't even lose any fighters because I neglected to launch them. 

Though I just realized the damn Shield Beetle had unique gear on it, because of course it did. Sigh. Time to try again. 

I do think the balance is quite iffy (the cannon was rather helpful at times, and I still died several times before getting this result, and have died several times subsequently. Plus, y'know, that earlier battle with the Wardragons), but cannot account for the level of difference. Though I noticed the Wanderers having two Hurricanes. I remember them only having one when I played through first, and wonder whether they took losses en route that I somehow missed. They seemed to have more ships left at the end of this run then I thought they did at the start of the last one. 

Also, any time I DO end up getting the enemy's attention, my ship gets disabled shockingly quickly. I can go to dead in seconds at the edge of the battle if enemy attention turns the wrong way. I am aware that the Emerald Sword has paltry shields, but given the number of shield disruptors floating around that doesn't matter much. Things are lethal before there's a chance to really do anything about it. 

Savegames are attached. 

[Melchior Jack.txt](https://github.com/endless-sky/endless-sky/files/10252856/Melchior.Jack.txt)
[Melchior Jack~Before Station Battle.txt](https://github.com/endless-sky/endless-sky/files/10252857/Melchior.Jack.Before.Station.Battle.txt)

Subsequent to those saves, reversing my ratio of shield disruptors to Slicers has helped immensely - one shield disruptor doesn't seem to do much, while two is much more effective. I'm still dying a lot, but enemy stuff is dying too. 

A pause function would be really, really handy here.

**Comment by SimulatedKnave** [comment]
Had another one where everybody died (so two in a row). I think a LOT is depending on which side gets reinforcements and how fast - this one a friendly 512 has shown up, and that seems to be making a lot of difference. 

Spoke too soon. Three in a row with everyone dead. 

In any case, the results seem to be pretty variable. Any large group of Kor Moreti ships seems to have a much easier time killing an equally sized group of opponents, which is fine except they also swarm like the Unfettered. 

...Also, one of the times my Dredgers ran away once I got blown up. Didn't know that was a thing they could do. 

The attached save is launching from the station. Full fleet, I think all the Wanderers are still alive, etc. 

[Melchior Jack~~previous-1.txt](https://github.com/endless-sky/endless-sky/files/10252912/Melchior.Jack.previous-1.txt)

**Comment by Hurleveur** [comment]
Just to be sure, you are on the latest continuous? Any balancing would have to happen there since heat was restored to its previous state recently

**Comment by SimulatedKnave** [comment]
I'd been playing through on Steam before moving to continuous. I think I'm basically through most existing content now, so was going to switch over in the near future. 

Sounds like I'll do that later today.

**Comment by kydot** [comment]
I agree with this, this mission is crazy hard. Even with a highly customised emerald sword I was getting absolutely f*cked up every single attempt.

**Comment by Hurleveur** [comment]
Except if you customized it with skylances that's honestly something I think you should expect - it doesnt deal or tank damage quickly enough to help your allies and the fighters (you used them I hope) cant hold forever
Indeed I think the main cannon does not go trough enemies like one could expect it to

It's actually less good than a Hurricane here, in my humble opinion

**Comment by kydot** [comment]
Maybe so but regardless, it's a very powerful late-game ship that should be able to handle this mission. I also edited the save file to spawn in four extra hurricanes and still lost so I'm fairly convinced that the hostile fleet is a little overtuned to say the least.

**Comment by yafflehk** [comment]
The only way I've ever played that mission is to take zero ships, arrive cloaked, let the wanderers get savaged, land and do the thing, take of cloaked and leave while the wanderers get their butts kicked again...

**Comment by Hurleveur** [comment]
Oh by all rights it is too difficult I'm just saying a lone ES is not that impressive - the issue is the ones who dont turn have a few of the bigger ships

**Comment by kydot** [comment]
> take of cloaked and leave while the wanderers get their butts kicked again...

You can't, you have to defeat the fleet. Removing that requirement is probably a pretty good way to balance this mission, especially if you make it obvious to the player that you either need to bring a cloaking device, a remnant ship, or a shitload of firepower. It would be cool to offer the player a choice -- either do it stealthily by yourself or go to a different planet to pick up a support fleet and fight it out.

**Comment by Hurleveur** [comment]
The Mereti are each a part of their collective, if they are infighting its to represent the strugge. If you dont help your side win it could fail.
That said how about extra garanteed reinforcements from the nearby systems instead of relying on random spawns? Or just remove one model 512

**Comment by TheGiraffe3** [comment]
Can this issue be closed with the merge of #10189.

**Comment by Quantumshark** [comment]
Closing as resolved by #10189.
